docs/contributing.rst

```diff
@@ -66,6 +66,20 @@ notebook as a base.
 .. _ACCESS-Hive Forum: https://forum.access-hive.org.au/
 
 
+Tips to making a good recipe
+----------------------------
+
+As researchers, we often develop code as part of our research and most of the time the main aim of code development is to find answers to questions that have not yet been explored. Reproducibility may not always be considered as part of the code development process. If this is the case, we are including some tips to writing reproducible code that can be easily followed by other users. 
```

Comment:
```suggestion
As researchers, we often develop code as part of our research and most of the time the main aim of code development is to find answers to questions that have not yet been explored. Reproducibility may not always be considered as part of the code development process or a priority. If this is something that resonates but, at the same time, you’d like to lean more towards making your code shareable and reproducible, we are including here some tips and good practices. 
```

Reply:
This has been edited in latest commit (98c7e64)